testing/bdd/features/timing/delay.feature

```diff
@@ -68,13 +68,16 @@ Feature: Delay of a document sequence
     | 00:00:05   |              |           |          |          |          | 00:00:10 | 00:00:15           |                  | 00:00:15          |                 | 00:00:15        |               |            |          |          |
```

Comment:
That is technically correct, yes, but I still want to check that the output document is right.